ext/web/02_timers.js

```diff
@@ -388,6 +389,8 @@ function unrefTimer(id) {
   core.unrefOp(timerInfo.promiseId);
 }
 
+internals.setTimeout = setTimeout;
```

Comment:
I think we don't need to expose it here - we can do `const setTimeout = globalThis.setTimeout` in `40_testing.js`